### ISSUE TITLE
Clarify how many TimeDimensions a DSD may have

### ISSUE THREAD

**Issue by glennmdt** [glennmdt/sdmx-ml#44]
Reference: https://github.com/sdmx3mdt/public-consultation/issues/36

The DSD structure should enforce a maximum of one time dimension.

There is some debate about whether SDMX 3.0 should support use cases involving DSDs with more than one time dimension. For now, however, the TWG has confirmed that DSDs should continue to allow for a maximum of one time dimension (TIME_PERIOD).

**Comment by jgager** [comment]
While the schema structure for dimension list
```
				<xs:choice maxOccurs="unbounded">
					<xs:element ref="Dimension"/>
					<xs:element ref="TimeDimension"/>
				</xs:choice>
```
might appear to allow for unbounded time dimensions, the component uniqueness constraint prevents this. It is necessary to do it this way in order to allow for the time dimension to be positioned anywhere in the dimension key. So no change is needed.

However, in reviewing the comments I did see that the dimension type attribute and its uniqueness constraint is no longer needed since measure dimensions have been removed. This has been taken out.
